pkg/apis/core/validation/validation_test.go

```diff
@@ -7292,17 +7292,17 @@ func TestValidateEphemeralContainers(t *testing.T) {
 	} {
 		var PodRestartPolicy core.RestartPolicy
 		PodRestartPolicy = "Never"
-		if errs := validateEphemeralContainers(ephemeralContainers, containers, initContainers, vols, nil, field.NewPath("ephemeralContainers"), PodValidationOptions{}, &PodRestartPolicy); len(errs) != 0 {
+		if errs := validateEphemeralContainers(ephemeralContainers, containers, initContainers, vols, nil, field.NewPath("ephemeralContainers"), PodValidationOptions{}, &PodRestartPolicy, false); len(errs) != 0 {
```

Comment:
I found bool flags for these type of functions always cause confusion in the long run. Can you add a named variable for this like:

```
noUserNamespace := false

and just pass that it in these tests? It is not clear what false means in this case. 
Even better would be having this function take a enum to make it clear what this means